LOCALAPP.md

```diff
@@ -95,10 +95,12 @@ These are some of the required steps, tips and tricks when it comes to running a
 3. Configuration of LocalTest   
     The LocalTest application acts as an emulator of the Altinn 3 platform services. It provides things like authentication, authorization and storage. Everything your apps will need to run locally.   
 
-    Settings:    
+    Settings (under `LocalPlatformSettings`):
     - `LocalTestingStorageBasePath` - The folder to which LocalTest will store instances and data being created during testing.
-    - `AppRepositoryBasePath` - The folder where LocalTest will look for apps and their files.
     - `LocalTestingStaticTestDataPath` - Test user data like profile, register and roles. (`<path to altinn-studio repo>/src/development/TestData/`)
+    - `LocalAppMode` - (default `file`) If set to `http`, LocalTest will find the active app configuration and policy.xml using apis exposed on `LocalAppUrl`. (note that this is a new setting needs to be added manually under `LocalPlatformSettings`, it might also require updates to altinn dependencies for your apps in order to support this functionality)
```

Comment:
Could we rather mention the specific version their app must run to support this functionality? I can't remember it of the top of my head, but I'm hoping you do.

Reply:
I'd love to see these descriptions in the same order as they appear in appsettings.json. 
In practice, that means placing `AppRepositoryBasePath` before `LocalTestingStorageBasePath`.

Reply:
I have the longer description [here](https://github.com/Altinn/altinn-studio/pull/7657/files/a17ad11485ca7fb46286e566806cad6e9c4d8063#diff-57c9b8378d42da16fb1a429635de574e869db5a9d5cf297783cf19409b8cf1a0R12-R25) for when things fail.

```suggestion
    - `LocalAppMode` - (default `file`) If set to `http`, LocalTest will find the active app configuration and policy.xml using apis exposed on `LocalAppUrl`. (note that this is a new setting needs to be added manually under `LocalPlatformSettings`, it might also require updates to altinn dependencies for your apps in order to support this functionality if your app is from before December 2021 and using nuget packages older than `4.21.0`)
```

Reply:
I did reorder them to make them read more clearly. Seems silly to have the `mode` config documented after one of the configs that is only used in one mode.

Reply:
I'm thinking we should document the modes separately, and only have people read info related to the solution they want to use. But we can fix this ourselves after the pr is merged.